unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUD/Tests/EditMode/SettingsPanelTests_EditMode.cs

```diff
@@ -0,0 +1,77 @@
+using DCL.SettingsPanelHUD;
+using DCL.SettingsPanelHUD.Sections;
+using DCL.SettingsPanelHUD.Widgets;
+using NSubstitute;
+using NUnit.Framework;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Tests
+{
+    public class SettingsPanelTests_EditMode
+    {
+        private SettingsPanelHUDController panelController;
+        private ISettingsSectionView newSectionView;
+        private ISettingsSectionController newSectionController;
+        private SettingsSectionModel newSectionConfig;
+
+        [SetUp]
+        public void SetUp()
+        {
+            panelController = new SettingsPanelHUDController();
+            newSectionView = Substitute.For<ISettingsSectionView>();
+            newSectionController = Substitute.For<ISettingsSectionController>();
+            newSectionConfig = new SettingsSectionModel(
+                Sprite.Create(new Texture2D(10, 10), new Rect(), new Vector2()),
+                "TestSection",
+                new SettingsButtonEntry(),
+                new SettingsSectionView(),
+                new SettingsSectionController(),
+                new SettingsWidgetList());
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            panelController.sections.Clear();
```

Comment:
I see that we create `Sprite` and `Textures` in the `SetUp` method. Please make sure they are freed by the `TearDown` stage.

Reply:
Fixed in this new PR (in order to avoid mix things between the part I and II): https://github.com/decentraland/explorer/pull/1784